skimage/exposure/tests/test_exposure.py

```diff
@@ -212,12 +212,12 @@ def test_adapthist_grayscale():
     img = rgb2gray(img)
     img = np.dstack((img, img, img))
     with expected_warnings(['precision loss|non-contiguous input']):
-        adapted = exposure.equalize_adapthist(img, 10, 9, clip_limit=0.01,
+        adapted = exposure.equalize_adapthist(img, (52, 64), clip_limit=0.01,
```

Comment:
No, in this case I was not able to get the exact same result, because the old method was poorly handling boundary pixels.  The difference was slight, and more *correct*, I'd say.